openmdao/solvers/nl_gauss_seidel.py

```diff
@@ -137,6 +151,46 @@ def solve(self, params, unknowns, resids, system, metadata=None):
             normval = resids.norm()
             u_norm = np.linalg.norm(unknowns.vec - unknowns_cache)
 
+            if self.options['use_aitken']: # If Aitken acceleration is enabled
```

Comment:
@Kenneth-T-Moore @kmarsteller I merged to the latest master and added a test that uses Sellar MDA. Please let me know if anything else is required. Thanks!